src/shell/lisa_shell

```diff
@@ -308,19 +319,38 @@ nosetests -v --nocapture --nologcapture \
 }
 
 function lisa-test {
-CMD=${1:-help}
+local iterations=0 # Means use default - see libs/utils/test.py
+local have_more_args=no
+
 echo
-case "x${CMD^^}" in
-"xHELP")
-	_lisa-test-usage
-	;;
-*)
-	_lisa-test $*
+while [[ $# -gt 0 ]]; do
+        case "$1" in
+                help|-h|--help)
+                        _lisa-test-usage
+                        return 0
+                        ;;
+                -n|--wload-iterations)
+                        iterations="$2"
+                        shift
+                        ;;
+                *)
+                        have_more_args=yes
```

Comment:
Yeah I did have a rationale for doing it this way but I can't remember what it was so it was probably nonsense :D